manager/container.go

```diff
@@ -270,6 +273,44 @@ func (c *containerData) updateLoad(newLoad uint64) {
 	}
 }
 
+func (c *containerData) updateSubcontainerStats(stats *info.ContainerStats, path string) (*info.ContainerStats, error) {
```

Comment:
Should this be recursive? Calling load stats on "/" should sum up stats on all existing containers.